scripts/run_third_party_size_check.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
```

Comment:
This file would no longer be necessary since you are calling the python file directly.

Reply:
Done (https://github.com/oppia/oppia/pull/5743/commits/f558a9004cf45bac8835ab9f95035633b5ce2820)